src/partials/id_verification_options.hbs

```diff
@@ -0,0 +1,65 @@
+<div class="card">
+  <div class="org-logo">
+    <img class="org-logo__image" src="{{logo}}" alt="Company Logo"/>
+  </div>
+  <form class="form form--spacing-lg" id="AuthnWidgetForm">
+    <div class="stack stack--small">
+      <h1 id="header" class="heading" data-id="heading">Verification Required</h1>
+      <h4 id="description" class="heading heading--4 text-block" style="margin:27px;margin-top:0;" data-id="heading">
+        Select a method to receive a web link on your mobile device to start the verification process, or select the QR Code button.
+      </h4>
+      <div id="errorBlock" class="feedback feedback--error" style="display: none; justify-content: center; margin: 15px;" data-id="feedback">
+        <span class="feedback__icon pingicon-error-triangle"></span>
```

Comment:
Will the >errors util not work for this page?

Reply:
Replaced errorBlock with >errors util.